CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## dev
 
 - Allow skipping maintenance tasks during list command
- 
```

Comment:
It should be located in `dev` section.

Reply:
Oh wrong button, should be request change 😅

Reply:
It seems like the changelog is very messy due to merge conflicts. I will open a new PR to tidy it up.